opentreemap/stormwater/views.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django.conf import settings
+from django.contrib.gis.geos import Point
+from django.contrib.gis.measure import D
+
+from stormwater.models import PolygonalMapFeature
+
+
+def polygon_for_point(request, instance):
+    lng = request.GET['lng']
+    lat = request.GET['lat']
+    point = Point(float(lng), float(lat), srid=4326)
+
+    features = PolygonalMapFeature.objects.distance(point)\
+        .filter(instance=instance)\
+        .filter(polygon__distance_lte=(point, D(m=settings.MAP_CLICK_RADIUS)))\
```

Comment:
I don't see MAP_CLICK_RADIUS defined as part of this PR.

In general, a click buffer is most effectively defined as a function of the current zoom level, since the distance-to-pixel ratio is cut in half each time you zoom in. Because we are only dealing with levels 15 through 19, it may be possible to choose a reasonable, single value.